pynxtools/dataconverter/readers/mpes/reader.py

```diff
@@ -198,20 +198,6 @@ def handle_h5_and_json_file(file_paths, objects):
                 f"but {file_path} does not match.",
             )
 
-        if not os.path.exists(file_path):
```

Comment:
If this if statement is removed then one needs to mention the entire data path which can be quite long.

Reply:
Yes, I agree that this may require to write out the full path for the testing files. However, I think it is odd for users when they get an error about some testing data path when they used a wrong file path which actually happened multiple times to me. Additionally, I think for the test files this is covered in the tests anyways and one should just run the tests to check to files. That way it isn't necessary to type the full path, too.

Reply:
Makes sense as it only effects the test data.